### PR TITLE
[Term Entry] PHP Arrays array_fill()

### PR DESCRIPTION
**Pull request by EspressoCat**
### Description

- Added array_fill() term for PHP
- Closes issue #2556 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+Title: 'array_fill()'
+Description: 'This function fills an array with values and returns a filled array.'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:3]:
```suggestion
Description: 'Fills an array with values and returns a filled array.'
```

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:7]:
```suggestion
```

Comment (caupolicandiaz) [reply content/php/concepts/arrays/terms/array-fill/array-fill.md:7]: subjects should only contain two items

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Syntax
+
+```pseudo
+array_fill($start_index, $number_elements, $values)
+```
+
+The `array_fill()` function has three required parameters:
+
+- `$start_index`: This parameter specifies the first `index` of the returned array.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:26]:
```suggestion
- `$start_index`: This parameter specifies the first index of the returned array.
```

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- `$number_elements`: This parameter refers to the `number` of elements to insert into the array.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:27]:
```suggestion
- `$number_elements`: This parameter refers to the number of elements to insert into the array.
```

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- `$values`: Specifies the `value` to be used to fill the array.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:28]:
```suggestion
- `$values`: Specifies the values to be used to fill the array.
```

File: content/php/concepts/arrays/terms/array-fill/array-fill.md (hunk 3 above)
Comment (caupolicandiaz) [reply content/php/concepts/arrays/terms/array-fill/array-fill.md:26]: generally elements that are being referenced in a code block such as a variable name or function should use backticks. Generic terms for elements do not have to be formatted.

File: content/php/concepts/arrays/terms/array-fill/array-fill.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+)
+```
+
+## Codebyte Example
+
+This example is runnable and uses the `array_fill()` function:
+
+```codebyte/php
+<?php
+
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-fill/array-fill.md:59]: remove empty lines in the code block

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@EspressoCat I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by EspressoCat** [comment]
Thanks for the feedback @caupolicandiaz I made the requested changes.

**Review by caupolicandiaz — APPROVED** [review]
@EspressoCat Looks good. I'm passing this on for a second review.

**Review by SSwiniarski — APPROVED** [review]
@EspressoCat I made one minor edit and it looks ok to merge.

**Comment by SSwiniarski** [comment]
@EspressoCat Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/arrays/array-fill

image

**Comment by EspressoCat** [comment]
Thank you for taking the time to check the code @SSwiniarski ;)

**Pull request merged**